configuration/cloudformation/cwl-s3-export.template

```diff
@@ -47,13 +47,13 @@
         "LambdaS3BucketNamePrefix": {
             "Description": "The prefix of the S3 bucket which contains Lambda package to be used for archiving CloudWath Logs to S3. Note that the name of the bucket must contain a region name suffix. The following is the valid backet name syntax: <bucket_name_prefix>.us-east-1.",
             "Type": "String",
-            "Default": "alertlogic-public-repo",
+            "Default": "imalertlogic-public-repo",
```

Comment:
Can we change this back to `alertlogic-public-repo`? Just to avoid confusion with new users or users who reinstall

Reply:
resolved thank you